File: iqMath/iqMath.cpp
#include <iostream> //Handles input/output operations (cout/cin)
#include <string> //Import the string class (mostly used within cin statements)
using namespace std;

void welcome(int caller);
void expression();

int main(){ 
	welcome(0);
	return 0;
}

void welcome(int caller){
	cout << "Welcome to iqMath!" << endl;
	cout << "What would you like to do?" << endl;
	cout << "1. Evaluate an expression." << endl;
	if (caller == 1)
		cout << "Please enter an integer that appears in the list above!" << endl;
	string response;
	cin >> response;
	if (response == "1") {
		expression();
	}
	else if (response == "2"){
		cout << "This is coming soon" << endl;
	}
	else {
		system("cls");
		welcome(1);
	}
}

void expression(){
	cout << "Enter the expression you would like to evaluate:" << endl;
	system("pause");
}

File: iqMath.py
"""
    file: iqMath.py
    author: Orens Xhagolli
    description: This does math! (Sorry for general description,
    I'm still not sure how far to take this)
"""

import evaluate
#import os #Optional if you want to use commands like "clear()"


def main(param = None):
    print("Welcome to iqMath!")
    print("Please choose what would you like to do next.")
    print("1. Evaluate an expression.")

    if param is not None: #Bad input case
        print ("Please enter an integer from the above list.")
        
    response = int(input("Enter number: ").strip())
    if response == 1:
        evaluate.main() #transfers control to the other file
    else:
        main(1) #bad input
